### DISCUSSION TITLE
Linux Wine Login Failure

### DISCUSSION THREAD

**Discussion by bearrrmann** [ppy/osu#16038] [Q&A] [answered]
Im running Kubuntu and Wine with osu stable and I can enter my username and password and then it says verification pending forever, it doesnt send me a verification or anything it just says that a verification is pending. I can send a log or whatever it is if you tell me how

**Comment by frenzibyte** [comment, marked as answer]
This repository is for osu!(lazer), not osu!(stable). Please seek help elsewhere.

**Reply by bearrrmann** [reply]
oh, sorry where do I go?

**Reply by bdach** [reply]
stable is not and never was officially supported on linux via wine.

**Comment by leofleeo** [comment]
To run osu! on linux please download the appimage of osu lazer on the releases page.
